ambry-router/src/main/java/com/github/ambry/router/NonBlockingRouterMetrics.java

```diff
@@ -236,6 +237,8 @@ public class NonBlockingRouterMetrics {
   public Gauge<Integer> operationControllerQueueSize;
   public Gauge<Integer> readQueuedQuotaResourceCount;
   public Gauge<Integer> writeQueuedQuotaResourceCount;
+  public Gauge<Boolean> delayedRequestsInQueue;
+  public Gauge<Boolean> outOfQuotaRequestsInQueue;
```

Comment:
why not just make these two metrics as counters and show the queue sizes?

Reply:
that's all good, I have two points to make
1. It seems like we don't really use Gauge<Boolean> anywhere, we only have one metric that uses boolean value as gauge, but it's not in ingraph, so can we change it to integer or long
2. let' use metricRegistry.gauge to register metric, instread of metricRegistry.register, since the former deal with duplicate metrics.